apps/dialtone-documentation/docs/design/typography/index.md

```diff
@@ -1,89 +1,729 @@
+<path d="M415 249.59V131.579H435.062L437.218 144.507H438.544C442.192 139.203 446.668 135.391 451.974 133.071C457.39 130.75 463.304 129.59 469.715 129.59C478.116 129.59 485.356 131.247 491.435 134.562C497.625 137.767 502.434 142.794 505.86 149.645C509.287 156.386 511 164.949 511 175.336C511 185.612 509.287 194.176 505.86 201.026C502.434 207.767 497.681 212.794 491.601 216.109C485.632 219.314 478.724 220.916 470.876 220.916C466.786 220.916 462.862 220.474 459.104 219.59C455.345 218.706 451.919 217.38 448.824 215.612C445.729 213.734 443.076 211.413 440.865 208.651H439.705V249.59H415ZM462.917 201.026C468.002 201.026 472.202 200.087 475.518 198.209C478.945 196.22 481.487 193.457 483.145 189.921C484.803 186.275 485.632 181.91 485.632 176.828V173.844C485.632 168.54 484.803 164.12 483.145 160.584C481.487 156.938 478.945 154.231 475.518 152.463C472.092 150.584 467.836 149.645 462.751 149.645C457.667 149.645 453.411 150.64 449.984 152.629C446.558 154.507 443.96 157.325 442.192 161.082C440.534 164.728 439.705 169.148 439.705 174.341V176.165C439.705 180.032 440.147 183.513 441.031 186.607C442.026 189.59 443.518 192.187 445.508 194.397C447.497 196.496 449.929 198.154 452.803 199.369C455.677 200.474 459.048 201.026 462.917 201.026Z" fill="var(--dt-color-purple-400)"/>
+</g>
+<rect x="0.25" y="0.839996" width="745.5" height="299.5" rx="13.75" stroke="black" stroke-opacity="0.1" stroke-width="0.5"/>
+<defs>
+<radialGradient id="paint0_radial_347_13445" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(-7.6808e-06 0.988048) rotate(21.9066) scale(804.058 599.926)">
+<stop stop-color="var(--dt-color-purple-300)"/>
+<stop offset="0.85474" stop-color="#6EBFF4" stop-opacity="0.7"/>
+<stop offset="1" stop-color="#4690D5" stop-opacity="0.5"/>
+</radialGradient>
+<clipPath id="clip0_347_13445">
+<rect y="0.589996" width="746" height="300" rx="14" fill="var(--dt-color-foreground-primary)"/>
+</clipPath>
+</defs>
+</svg>
+
+## Introduction
+
+Typography is more than just selecting fonts; it's about creating a harmonious and effective reading experience. In this documentation, we'll explore how our chosen typefaces, styles, and layouts come together to convey a consistent brand identity across digital interfaces.
+
+## Core Principles
+
+Understanding the fundamentals is key. Learn about the anatomy of type, the interplay of typefaces, and how to strike the right balance between readability and aesthetics. Discover how size, spacing, and hierarchy contribute to a seamless user experience.
+
+## Typeface selection
+
+To define the Typeface selection, we take the accessibility is a priority to us, so we approach it by using the default system fonts by each operating system to have the best conversion to all possible variables of our product (Japanese, Russian, etc)
```

Comment:
"we take the accessibility is a priority to us" Something wrong with the wording here.. Did we mean to say "we make accessibility a priority"?

Reply:
great! i think the autocorrect somehow change 'make' to 'take', but I like your approach, i think is more straight forward, updating that!